funnel/views/project.py

```diff
@@ -577,6 +578,9 @@ def register(self):
                 rsvp.rsvp_yes()
                 db.session.commit()
                 flash(_("You have successfully registered"), 'success')
+                user_registered_for_project.send(
+                    self.obj, user=current_auth.user, locale=get_locale()
```

Comment:
Move this `get_locale` call to the signal receiver. It's not the view's problem. The signal receiver is the one triggering a background job that needs the locale.